src/Game.js

```diff
@@ -42,9 +47,9 @@ class Game {
     return this.players;
```

Comment:
We could refactor this a bit to do a loop through our `playerInput` array and instantiate a new player for each input in that array. Some duplicate code here that could be DRYed up with a loop :)